concordium-sdk-examples/src/main/java/com/concordium/sdk/examples/contractexample/cis2nft/Cis2NftMethod.java

```diff
@@ -0,0 +1,16 @@
+package com.concordium.sdk.examples.contractexample.cis2nft;
+
+/**
+ * Represents the different possible methods able to be run by {@link Cis2Nft}
+ */
+public enum Cis2NftMethod {
```

Comment:
Link to the source code (and document what the different values mean)

Reply:
How in-depth should the explanation of the values be? Perhaps i should link the parameters each value is used with?

Reply:
<img width="461" alt="image" src="https://github.com/Concordium/concordium-java-sdk/assets/93247743/cbc6e3ec-0768-4e84-86f7-ce216bf46db3">

Reply:
This is OK, but please provide a link to the source. Then users can look up the finer details themselves.